RecoCTPPS/TotemRPLocal/src/TotemTimingRecHitProducerAlgorithm.cc

```diff
@@ -191,5 +191,5 @@ float TotemTimingRecHitProducerAlgorithm::constantFractionDiscriminator(const st
                        (dataProcessed[indexOfThresholdCrossing - 1] - dataProcessed[indexOfThresholdCrossing]) *
                        (threshold - dataProcessed[indexOfThresholdCrossing]) +
                    time[indexOfThresholdCrossing]
-             : TotemTimingRecHit::NO_T_AVAILABLE;
+             : (int)TotemTimingRecHit::NO_T_AVAILABLE;
```

Comment:
It must return a ```float```

Reply:
I'll change it, yet:
https://github.com/cms-sw/cmssw/blob/master/DataFormats/CTPPSReco/interface/TotemTimingRecHit.h#L20